lib/option.rb

```diff
@@ -48,17 +48,20 @@ def self.postgres_locations
     ["almalinux-9", "AlmaLinux 9"]
   ].map { |args| BootImage.new(*args) }.freeze
 
-  VmSize = Struct.new(:name, :family, :cores, :vcpus, :memory_gib, :storage_size_options, :visible, :gpu, :arch) do
+  IoLimits = Struct.new(:max_ios_per_sec, :max_read_mbytes_per_sec, :max_write_mbytes_per_sec)
+  NO_IO_LIMITS = IoLimits.new(nil, nil, nil)
```

Comment:
```suggestion
  NO_IO_LIMITS = IoLimits.new(nil, nil, nil).freeze
```

You generally want constant values to be frozen.  In this case, it is important, as the object is shared by multiple (all current?) VmSizes?